sources/assets/exegol/load_supported_setups.sh

```diff
@@ -52,6 +52,14 @@ function deploy_vim() {
   fi
 }
 
+function deploy_nvim () {
+  #### neovim deployment
+  if [ -d "$MY_Setup_PATH/nvim" ]; then
```

Comment:
Better if we replace `$MY_Setup_PATH` with `$MY_SETUP_PATH` @ShutdownRepo ?

Reply:
Agreed, but let’s take care of that in another PR as those variables are set all across the file